src/components/ui/check-chip-group.svelte

```diff
@@ -0,0 +1,70 @@
+<script>
+  import { createEventDispatcher } from 'svelte';
+
+  export let classname = '';
+  export let chipwrapperclass = '';
+  export let chipclass = '';
+  export let inputclass = '';
+  export let small = false;
+  export let outline = false;
+
+  export let values;
+  export let labels = null;
+  export let name;
+  export let maxChecked = null;
+  let currentChecked = 0;
+  values.forEach((elt) => currentChecked += elt.checked);
```

Comment:
Don't see why it shouldn't work. Keep in mind that in the UI kit, two sets of chips are bound to the same value array, so if you updated the array through one set of chips, the other one has an incorrect value. This is not intended behaviour, however, to do such bindings, so inc/dec should be fine in the actual use cases. And more efficient

Reply:
It seems that the shared array is indeed the reason for this problem, but it doesn't come up with the reactive solution. It may have to do with the fact that I'm currently learning Haskell, but I still believe that the functional solution is better/more elegant)

Reply:
Also, does efficiency really matter here when the array is almost never bigger than 10 elements? The functional/reactive solution will help avoid any such problems by simply defining what a variable describes instead of how to mutate it

Reply:
Eh. Fine